lightning/src/ln/channelmanager.rs

```diff
@@ -4101,6 +4101,32 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		}
 	}
 
+	/// Called to respond to a request to open a channel after the
+	/// [`events::Event::OpenChannelRequest`] has been triggered.
+	///
+	/// If `accept_channel` is set to true, the requested channel will be opened. If
+	/// `accept_channel` is false, the open channel request will be rejected.
+	///
+	/// The `temporary_channel_id` parameter indicates which channel the reponse is for.
+	///
+	/// [`events::Event::OpenChannelRequest`]: crate::util::events::Event::OpenChannelRequest
+	pub fn respond_to_open_channel_request(&self, accept_channel: bool, temporary_channel_id: [u8; 32]) -> Result<(), APIError> {
```

Comment:
nit: I'd kinda prefer an API of `accept_inbound_channel` with users instead calling `force_close_channel` to reject the channel, that way our code stays consistent and we use the same codepaths for everything.

Reply:
That makes a lot of sense! I'll change to that.